doc/developers-guide/OpenSSL-and-FIPS.md

```diff
@@ -10,24 +10,25 @@ Currently known incompatible features are:
 
 ## Requirements
 
-### Custom Erlang/OTP 17.4.1 with FIPS support
+### build Erlang/OTP FIPS support
 
-Compile Erlang/OTP from [michalwski/otp 17.4.1-fips](https://github.com/michalwski/otp/tree/fips-17.4.1).
 Make sure the option `--enable-fips` is specified for `configure` command.
 If you want to use a different OpenSSL than the default one, specify the option `--with-ssl=PATH_TO_YOUR_OPENSSL` as well.
 Here's an example of a command for building Erlang/OTP with kerl:
 ```
-KERL_CONFIGURE_OPTIONS="--enable-fips --with-ssl=/home/vagrant/openssl" \
-./kerl build git https://github.com/michalwski/otp.git fips-17.4.1 17.4.1-fips
+KERL_CONFIGURE_OPTIONS="--enable-fips" ./kerl build 20.0 20.0-fips
 ```
 
 ### Building MongooseIM with a custom OpenSSL
 
-Before running `./rebar3 compile` or `make rel` please export the CFLAGS and LDFLAGS env vars pointing to a FIPS compliant OpenSSL, f.e.
+If you want to use a different OpenSSL than the default one, before running `./rebar3 compile` or `make rel` please export the CFLAGS and LDFLAGS env vars pointing to a FIPS compliant OpenSSL, f.e.
```

Comment:
If you want to use a custom OpenSSL, please export the CFLAGS and LDFLAGS env vars pointing to a FIPS compliant OpenSSL before running `./rebar3 compile` or `make rel`.
F.e: